### PR TITLE
Fix/warnings

### PR DESCRIPTION
**Pull request by TheMaskedGuy**

### CODE REVIEW

File: lib/src/widgets/gs_input/gs_input.dart [hunk 1]
```diff
@@ -286,13 +286,13 @@ class _GSInputState extends State<GSInput> {
       return styler.onFocus?.borderColor;
     }
 
-    double? resolveFocusBorderWidth() {
-      if (inputVariant == GSVariants.underlined) {
-        return styler.onFocus?.borderBottomWidth ?? styler.borderBottomWidth;
-      }
+    // double? resolveFocusBorderWidth() {
+    //   if (inputVariant == GSVariants.underlined) {
```
Comment (geekruchika) [inline lib/src/widgets/gs_input/gs_input.dart:290]: remove commented code

File: lib/src/widgets/gs_text_area/gs_text_area.dart [hunk 2]
```diff
@@ -259,16 +259,16 @@ class _GSTextAreaState extends State<GSTextArea> {
       return styler.onFocus?.borderColor;
     }
 
-    double? resolveFocusBorderWidth() {
-      return styler.onFocus?.borderWidth ?? styler.borderWidth;
-    }
+    // double? resolveFocusBorderWidth() {
+    //   return styler.onFocus?.borderWidth ?? styler.borderWidth;
```
Comment (geekruchika) [inline lib/src/widgets/gs_text_area/gs_text_area.dart:263]: remove

File: lib/src/widgets/gs_text_area/gs_text_area.dart [hunk 3]
```diff
@@ -259,16 +259,16 @@ class _GSTextAreaState extends State<GSTextArea> {
...
+    // }
 
     final borderColor = resolveBorderColor();
     final borderWidth = resolveBorderWidth();
     final focusedBorderColor = resolveFocusBorderColor();
     // final focusedBorderWidth = resolveFocusBorderWidth();
-    final hintStyle =
-        widget.hintStyle ?? styler.descendantStyles?['_input']?.textStyle;
+    // final hintStyle =
+    //     widget.hintStyle ?? styler.descendantStyles?['_input']?.textStyle;
```
Comment (geekruchika) [inline lib/src/widgets/gs_text_area/gs_text_area.dart:271]: remove

**Review by geekruchika — CHANGES_REQUESTED** [review]

File: lib/src/widgets/gs_text_area/gs_text_area.dart (hunk 3 above)
Comment (TheMaskedGuy) [reply lib/src/widgets/gs_text_area/gs_text_area.dart]: Resolved in latest commit.

File: lib/src/widgets/gs_text_area/gs_text_area.dart (hunk 2 above)
Comment (TheMaskedGuy) [reply lib/src/widgets/gs_text_area/gs_text_area.dart]: Resolved in latest commit.

File: lib/src/widgets/gs_input/gs_input.dart (hunk 1 above)
Comment (TheMaskedGuy) [reply lib/src/widgets/gs_input/gs_input.dart]: Resolved in latest commit.

**Pull request merged**